packages/calcite-components/src/components/radio-button/radio-button.scss

```diff
@@ -29,7 +28,7 @@
     all var(--calcite-animation-timing) ease-in-out 0s,
     outline 0s,
     outline-offset 0s;
-  box-shadow: var(--calcite-radio-button-shadow, inset 0 0 0 1px var(--calcite-color-border-input));
+  box-shadow: inset 0 0 0 1px var(--calcite-radio-button-border-color, var(--calcite-color-border-input));
```

Comment:
nice catch with border color. I think we could clean up this file a bit by using internal variables

```
box-shadow: inset 0 0 0 var(--calcite-internal-radio-button-shadow-spread, 1px)
    var(
      --calcite-radio-button-border-color,
      var(--calcite-internal-radio-button-border-color, var(--calcite-color-border-input))
    );
```

Reply:
@alisonailea Cleaned this up a bit. Would you mind taking another look?